arcsi/api/tag.py

```diff
@@ -0,0 +1,23 @@
+from marshmallow import fields, post_load, Schema
+
+from arcsi.api import arcsi
+from arcsi.model.tag import Tag
+
+
+class TagDetailsSchema(Schema):
+    id = fields.Int()
+    display_name = fields.Str(required=True)
+    
+    @post_load
+    def make_tag(self, data, **kwargs):
+        return Tag(**data)
+
+tags_details_schema = TagDetailsSchema()
+tags_partial_schema = TagDetailsSchema(partial=True)
+many_tags_schema = TagDetailsSchema(many=True)
+
+@arcsi.route("/tags", methods=["GET"])
```

Comment:
@pvj nice demo API! But why demo? Wouldn't we need it for example to list all shows with the same tag (a feature we have in the current website)? And for that wouldn't it be easier that the API returns the secondary relations?

Reply:
@gammaw "Demo" in the sense that it just lists the existing tags from the database for proof of concept that the API works. 

Now I'll add the tag details page that will return those relations and then I'll consider it more than a demo